src/frontend/komponenter/Fagsak/Vilkårsvurdering/VilkårsvurderingSkjema.tsx

```diff
@@ -109,6 +116,20 @@ const VilkårsvurderingSkjema: React.FunctionComponent<IVilkårsvurderingSkjema>
                         </PersonLinje>
 
                         <Collapse isOpened={personErEkspandert[personResultat.personIdent]}>
+                            {skalViseRegisteropplysninger && (
+                                <>
+                                    {personResultat.person.registerhistorikk ? (
+                                        <Registeropplysninger
+                                            opplysninger={personResultat.person.registerhistorikk}
+                                        />
+                                    ) : (
+                                        <AlertStripe
+                                            type={'advarsel'}
+                                            children={'Klarte ikke hente registeropplysninger'}
```

Comment:
Kan det være bedre å si noe annet enn "klarte ikke". Er dette tilfellene hvor vi har en gammel behandling f.eks.? Isåfall så vet vi jo at det ikke finnes noe :)

Reply:
Nei, som det står på screenshot i pr så er dette hvis noe feiler. Vi bruker "klarte ikke" til lignende caser nå, men kan endre hvis du har forslag? :) 
 
Screenshot 1 er caset du nevner

Reply:
og de skal fikse tekst der